### ISSUE TITLE
Angular animate version conflict with bower install

### ISSUE THREAD

**Issue by ionparticle** [ubc/acj-versus#212]
Which fails gulp, I'm not sure which version is correct, but the dependency resolution should be stored into bower.json. 

Needed for upgrade on prod.

**Comment by ionparticle** [comment]
Looks like the correct version is 1.3.4

**Comment by mwytang** [comment]
I didn't have any problem upgrading from master to develop. What is conflicting with what?

**Comment by ionparticle** [comment]
```
Unable to find a suitable version for angular-animate, please choose one:
    1) angular-animate#~1.2.10 which resolved to 1.2.28 and is required by angular-motion#0.3.3 
    2) angular-animate#1.3.4 which resolved to 1.3.4 and is required by acj 
    3) angular-animate#>1.2.8 which resolved to 1.3.13 and is required by AngularJS-Toaster#0.4.9Prefix the choice with ! to persist it to bower.json
```

Basically, AngularJS-Toaster is fine with any version >1.2.8, which makes it get the latest Angular-animate 1.3.13. This causes problems cause our version of Angular is still 1.3.4.

**Comment by ionparticle** [comment]
Confirm fixed.
